frontend/libs/studio-components-legacy/src/components/StudioSectionHeader/StudioSectionHeader.tsx

```diff
@@ -32,7 +31,7 @@ const StudioSectionHeader = forwardRef<HTMLDivElement, StudioSectionHeaderProps>
             {heading.text}
           </Heading>
         </div>
-        {helpText && <StudioHelpText aria-label={helpText.title}>{helpText.text}</StudioHelpText>}
+        {helpText && <HelpText title={helpText.title}>{helpText.text}</HelpText>}
```

Comment:
I have changed the attribute because `title` is required on `HelpText`. It's also better to use `title` than `aria-label`.